.github/workflows/CI-PR.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   build:
     name: Test and build
-    runs-on: macOS-10.14    
+    runs-on: macOS-latest
```

Comment:
Isn't there a chance of checks failing on PRs because of problems unrelated to the changes in it, if CI automatically switches to the latest macOS?

Reply:
Perhaps, but as per [GitHub guidance](https://github.blog/changelog/2019-11-06-github-actions-macos-virtual-environment-updated-to-catalina/) we need to move to `macOS-latest` 

> The GitHub Actions macOS virtual environment has been upgraded to Catalina (v10.15). Jobs using the `macos-10.14` virtual environment will not run and must be migrated to use `macos-latest`.

Reply:
Ahh, I see.